### DISCUSSION TITLE
配置的邮箱测试ok，但表单收集的通知邮件收不到

### DISCUSSION THREAD

**Discussion by lomiles** [halo-dev/halo#5919] [General]
版本2.15.2
![image](https://github.com/halo-dev/halo/assets/35911812/42b8b2b8-820c-4fe1-a905-9272782ca899)
表单的设置如下
![image](https://github.com/halo-dev/halo/assets/35911812/2f4728ca-a63e-4c03-a2a1-7fcca1933820)

**Comment by LIlGG** [comment]
需要配置的是用户名，在个人资料中可以看到，配置邮箱是无效的。

**Comment by lomiles** [comment]
可以了，用户名所用的邮箱也要先验证通过
